src/ivcam/sr300.cpp

```diff
@@ -457,14 +461,13 @@ namespace librealsense
         _hw_monitor->get_gvd(gvd_buff.size(), gvd_buff.data(), GVD);
 
         auto fw_version = _hw_monitor->get_firmware_version_string(gvd_buff, fw_version_offset);
+        auto recommended_fw_version = firmware_version(SR3XX_RECOMMENDED_FIRMWARE_VERSION);
```

Comment:
This attribute is fluid and shall be managed per SKU. I believe that's the reason it was originally disabled